src/Stratis.Bitcoin.Features.Wallet/WalletManager.cs

```diff
@@ -219,7 +219,7 @@ public Mnemonic CreateWallet(string password, string name, string passphrase = n
 
             this.logger.LogTrace("({0}:'{1}')", nameof(name), name);
 
-            // For now the passphrase is set to be the password by default.
+            // For now the passphrase is set to be the password by default, if not supplied. This should only happen if passphrase is null, not when it is empty string (allowed value).
```

Comment:
After a chat wit @bokobza I agree with him we should separate password from passphrase (at the expense of backward compatibility) a user restoring an old wallet will have to provide the passphrase in the correct field.

Reply:
Great, explicit is the best =)

Should I update the PR, and make passphrase a required field, but allowing empty strings. For both Create and Restore?

```
[Required(ErrorMessage = "A passphrase is required.", AllowEmptyStrings = true)]
```

And then remove the logic that copies password to passphrase.

Reply:
yes please. This way, users using the latest version will know they need to provide support for the passphrase or at least let their users know of the change.

Reply:
To make sure it is explicit in both REST API and C# library, I removed the default value (null) for the parameter. This means I had to touch a lot of unit test files, but I think that is the best way to go about this. Doing final testing and then I'll update the PR, might do some merge if there has been a lot of changes recently.

Reply:
> This means I had to touch a lot of unit test files, but I think that is the best way to go about this.

Absolutely.